src/store.js

```diff
@@ -0,0 +1,28 @@
+export default class Store {
+  // Get Todos from Local Storage
+  static getTodos() {
+    const todos = JSON.parse(localStorage.getItem('todos')) || [];
+    return todos;
+  }
+
+  static add(task) {
+  // Add a Todo to Local Storage
+    const tasks = Store.getTodos();
+    tasks.push(task);
+    localStorage.setItem('todos', JSON.stringify(tasks));
+  }
+
+  static setTodos(todos) {
+    localStorage.setItem('todos', JSON.stringify(todos));
+  }
+
```

Comment:
- [ ]  Make sure that all changes are directly being saved in your local storage. For me to update the item in the browser, I will need to reload the local storage so that I can see the changes in the local storage. 
![image](https://user-images.githubusercontent.com/43877689/177588859-75307c55-fbcf-4d13-aa74-c7aedf721403.png)

Reply:
Hey @ishimwezachee ,

I'm sorry but could you clarify on the Local Storage Updating...
Because the tasks will not change when reloaded if they are not stored locally

Charles.